core/chains/evm/types/models.go

```diff
@@ -52,16 +52,25 @@ var _ commontypes.Head[common.Hash] = &Head{}
 var _ htrktypes.Head[common.Hash, *big.Int] = &Head{}
 
 // NewHead returns a Head instance.
-func NewHead(number *big.Int, blockHash common.Hash, parentHash common.Hash, timestamp uint64, chainID *ubig.Big) Head {
+func NewHead(number *big.Int, blockHash common.Hash, parentHash common.Hash, chainID *ubig.Big) Head {
```

Comment:
Feels like this ought to be in test code really since I can't imagine when we'd use this in real/prod

Reply:
Yes, it only has test usages so it can move to `package testutils`, but how about as a follow-up?